scripts/embassy.ts

```diff
@@ -0,0 +1,4 @@
+//export { setConfig } from "./services/setConfig.ts";
+//export { getConfig } from "./services/getConfig.ts";
+//export { properties } from "./services/properties.ts";
+//export { migration } from "./services/migrations.ts";
```

Comment:
```suggestion
// This file is commented out because it is not currently used for this service. This exists merely as an example on how to set up the configuration. 
//export { setConfig } from "./services/setConfig.ts";
//export { getConfig } from "./services/getConfig.ts";
//export { properties } from "./services/properties.ts";
//export { migration } from "./services/migrations.ts";
```